src/features/landing/components/ForWhoSection/ForWhoSection.stories.tsx

```
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Meta, StoryObj } from '@storybook/react';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)